lib/ruby-debug-ide/xml_printer.rb

```diff
@@ -389,9 +392,19 @@ def inspect_with_allocation_control(slice, memory_limit)
       result = nil
       inspect_thread = DebugThread.start {
         start_alloc_size = ObjectSpace.memsize_of_all
```

Comment:
I sill getting an INTERNAL ERROR (pipe or something)  when using probabilistic scheme. Apparently, it is better to abandon this proposal.